src/jobservice/job/impl/gcreadonly/garbage_collection.go

```diff
@@ -288,9 +288,8 @@ func (gc *GarbageCollector) removeUntaggedBlobs(ctx job.Context) {
 			continue
 		}
 		p := result.Data
-		all, err := gc.blobMgr.List(ctx.SystemContext(), blob.ListParams{
-			ProjectID: p.ProjectID,
-		})
+
+		all, err := gc.blobMgr.List(ctx.SystemContext(), q.New(q.KeyWords{"projectID": p.ProjectID}))
```

Comment:
Same processing as the GC task?

Reply:
this will be deprecated, no need to update.